benchmarkapps/Benchmarks/Program.cs

```diff
@@ -29,22 +29,22 @@ public static IWebHostBuilder GetWebHostBuilder(string[] args)
                     .UseKestrel();
 
             var scenario = config["scenarios"]?.ToLower();
-            if (scenario == "plaintextdispatcher")
+            if (scenario == "plaintextdispatcher" || scenario == "plaintextglobalrouting")
```

Comment:
I've added new names that reflect the correct branding. I need to talk to some folks about what do to change what the perf server does. I plan to follow up and remove the old names after updating the perf automation.